skbeam/core/correlation.py

```diff
@@ -769,3 +769,29 @@ def _validate_and_transform_inputs(num_bufs, num_levels, labels):
 
     return (label_array, pixel_list, num_rois, num_pixels,
             lag_steps, buf, img_per_level, track_level, cur)
+
+
+def one_time_from_two_time(two_time_corr):
+    """
+    This will provide the one-time correlation data from two-time correlation
+    data. 
+
+    Parameters
+    ----------
+    two_time_corr : array
+        matrix of two time correlation
+        shape (number of images, number of images, number of labels(ROI))
```

Comment:
I think your shape is backwards. Shouldn't this be `(num ROIs, num images, num images)`?  Same below in the `Returns` section